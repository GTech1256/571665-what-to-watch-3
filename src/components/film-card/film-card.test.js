import React from "react";
import renderer from "react-test-renderer";
import FilmCard from "./film-card.jsx";

const mockData = {
  name: `Fantastic Beasts`
};

it(`<FilmCard /> should render correct`, () => {
  const tree = renderer
    .create(<FilmCard
      name={mockData.name}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
})
;
