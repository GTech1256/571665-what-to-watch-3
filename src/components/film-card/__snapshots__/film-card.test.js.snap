// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmCard /> should render correct 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height="175"
      poster="img/the-grand-budapest-hotel-poster.jpg"
      src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Fantastic Beasts
    </a>
  </h3>
</article>
`;
